scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -123,6 +123,26 @@ trait ClassLikeSupport:
           .map { _ =>
             parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))
           }
+          
+      case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isExported =>
+        val exportedTarget = dd.rhs.flatMap { 
+          case a: Apply => Some(a.fun)
+          case s: Select => Some(s)
+          case _ => None
+        }.map { 
+          case s: Select => s
+        } 
+        val functionName = exportedTarget.fold("instance")(_.name)
+        val instanceName = exportedTarget.fold("function")(_.qualifier.asInstanceOf[Select].name)
+        val dri = dd.rhs.flatMap {
+          case a: Apply => None
+          case s: Select => 
+            val dri = s.symbol.dri 
+            dri.getCallable match
+              case null => None
+              case _ => Some(dri)
+        }.orElse(exportedTarget.map(_.qualifier.tpe.typeSymbol.dri))
```

Comment:
Will `typeSymbol` works with object? See comment above.

Reply:
Hard to tell. Could not produce anything breaking the generation. However, there is one interesting thing, for that code:
```scala
object X:
  def x: Int = 1
  val x2: Int = 1
  var x3: Int = 1

class B:
  val a = new A
  export a._
  export X._
```

we get following output:

![obraz](https://user-images.githubusercontent.com/32793002/100462880-3627bb80-30cb-11eb-82b3-91ecc5599814.png)

I will have to check is it our bug, but it sees `x2` as `Int`, not literal `1`, which is odd, since that type is stable and safe to assume it's literal type (as it did with previous `vals`)